docs/tutorial/offscreen-rendering.md

```diff
@@ -40,22 +40,22 @@ To enable this mode, GPU acceleration has to be disabled by calling the
 ## Example
 
 ```javascript fiddle='docs/fiddles/features/offscreen-rendering'
-const { app, BrowserWindow } = require('electron')
-const fs = require('fs')
+const { app, BrowserWindow } = require("electron");
+const fs = require("fs");
 
-app.disableHardwareAcceleration()
+app.disableHardwareAcceleration();
 
-let win
+let win;
 
 app.whenReady().then(() => {
-  win = new BrowserWindow({ webPreferences: { offscreen: true } })
-
-  win.loadURL('https://github.com')
-  win.webContents.on('paint', (event, dirty, image) => {
-    fs.writeFileSync('ex.png', image.toPNG())
-  })
-  win.webContents.setFrameRate(60)
-})
+  win = new BrowserWindow({ webPreferences: { offscreen: true } });
+
+  win.loadURL("https://github.com");
+  win.webContents.on("paint", (event, dirty, image) => {
+    fs.writeFileSync("ex.png", image.toPNG());
+  });
+  win.webContents.setFrameRate(60);
+});
```

Comment:
I'm surprised this passed linting, since all of the other examples are standardjs, not prettier.

Reply:
Hmm, I think this was a mistake. I meant to just change the link, but I think this got applied by Format on Save.